src/status_im/ui/screens/profile/tribute_to_talk/views.cljs

```diff
@@ -1,8 +1,10 @@
+
```

Comment:
how's this related to onboarding?

Reply:
Leftover from numpad migration to separate PR, removed.

Reply:
still here

Reply:
Removed for good :)